.gitignore

```diff
@@ -1,3 +1,8 @@
+openduck-py/*.wav
```

Comment:
We have some .wav files in source control already that we need

Reply:
this will only ignore those that are in the top level openduck-py directory - i was saving some stuff there so made sense to add this